confidant/routes/credentials.py

```diff
@@ -160,7 +161,7 @@ def get_credential(id):
     metadata_only = request.args.get(
         'metadata_only',
         default=False,
-        type=bool,
+        type=inputs.boolean,
```

Comment:
See https://github.com/flask-restful/flask-restful/issues/397.  

Oddly, I don't remember running into this issue when using `reqparse` so I assume it's just because I'm using `request.args.get`.  Either way, the bug is definitely fixed now

Reply:
I don't see how this is an issue. We're not seeing this for the get_service endpoint.

Reply:
Spoke offline - moving to a helper function